internal/convert/orca.go

```diff
@@ -42,15 +42,15 @@ func getDefaults(h *config.Hal) *config.Orca_Defaults {
 
 func getPipelineTemplates(h *config.Hal) *config.Orca_PipelineTemplates {
 	// TODO: Consider whether this should actually be a BoolWrapper so that we
-	// can distinguish an explicitly set 'false' from an unset value
-	if !h.GetFeatures().GetPipelineTemplates() {
+	// can distinguish an explicitly set 'wrappers.False()' from an unset value
```

Comment:
Do we still need this TODO?

Reply:
Oh, good call, I'll add a TODO to remove it.

Reply:
(Or will actually remove it...)